vm/src/frame.rs

```diff
@@ -1170,12 +1173,9 @@ impl Frame {
     }
 
     fn pop_multiple(&self, count: usize) -> Vec<PyObjectRef> {
-        let mut objs: Vec<PyObjectRef> = Vec::new();
-        for _x in 0..count {
-            objs.push(self.pop_value());
-        }
-        objs.reverse();
-        objs
+        let mut stack = self.stack.borrow_mut();
+        let stack_len = stack.len();
+        stack.drain(stack_len - count..stack_len).collect()
```

Comment:
I'm not sure whether this makes the code more readable. The previous range from `0..count` was clearer in my opinion. I would rather revert this change.

Reply:
This is more efficient, it preallocates a Vec to put the elements into and it doesn't have to pop backwards and then reverse().